vendor/templates/vendor/manage/subscription_detail.html

```diff
@@ -0,0 +1,164 @@
+            {{payment.payee_full_name}}
+          </div>
+          <div class='d-flex flex-column w-50 px-3 py-2'>
+            <div class='font-weight-bold'>{% trans 'Account' %}</div>
+            {{payment.result.account_number}}
+          </div>
+          <div class='d-flex flex-column w-100 px-3 py-2'>
+            <div class='font-weight-bold'>{% trans 'Billing Address:' %}</div>
+            {{payment.billing_address.get_address_display}}
+          </div>
+          <div class='d-flex flex-column w-100 px-3 py-2'>
+            <div class='font-weight-bold'>{% trans 'Actions:' %}</div>
+            <div class='d-flex flex-row'>
+              <form class="mr-3">
+                <a class="btn btn-sm btn-primary" data-toggle="modal" data-target="#update_billing_info">
+                  {% trans 'Update Card' %}
+                </a>
+              </form>
+              <form>
+                <a class="btn btn-sm btn-primary" data-toggle="modal" data-target="#update_billing_address">
+                  {% trans 'Update Address' %}
+                </a>
+              </form>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+  <div class='modal fade' tabindex="1" role="dialog" id="update_billing_info">
+    <div class='modal-dialog modal-dialog-centered' role="document">
+      <div class='modal-content p-5'>
+        <h3 class="mb-3">{% trans 'Update Card Info' %}</h3>
+        <hr class="m-0" />
+        <div class='d-flex flex-row w-100 align-items-center my-3'>
+          <form method="POST" action="{% url 'vendor:customer-subscription-update-payment' object.uuid %}">
+            {% csrf_token %}
+            {{ payment_form.as_p }}
+            <button class="btn btn-primary mt-3">{% trans 'Update' %}</button>
+            <button class="btn btn-link mt-3" type="button" role="button" aria-pressed="true" data-toggle="modal"
+              data-target="#update_billing_info">{% trans 'Cancel' %}</button>
+          </form>
+        </div>
+      </div>
+    </div>
+  </div>
+  <div class='modal fade' tabindex="1" role="dialog" id="update_billing_address">
+    <div class='modal-dialog modal-dialog-centered' role="document">
+      <div class='modal-content p-5'>
+        <h3 class="mb-3">{% trans 'Update Billing Address' %}</h3>
+        <hr class="m-0" />
+        <div class='d-flex flex-row w-100 align-items-center my-3'>
+          <form method="POST" action="{% url 'vendor:customer-billing-update' billing_form.instance.uuid %}">
+            {% csrf_token %}
+            {{ billing_form.as_p }}
+            <button class="btn btn-primary mt-3">{% trans 'Update' %}</button>
+            <button class="btn btn-link mt-3" type="button" role="button" aria-pressed="true" data-toggle="modal"
+              data-target="#update_billing_address">{% trans 'Cancel' %}</button>
+          </form>
+        </div>
+      </div>
+    </div>
+  </div>
+</div>
+{% endblock %}
```

Comment:
@rhimmelbauer Are there designs for these templates by chance?

Reply:
No design comps for this at the moment. this is just to enable and display certain functionality and test it